dart/whatsapp_with_vonage/lib/utils.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+import 'package:path/path.dart' as p;
+
+final staticFolder = p.join(p.dirname(Platform.script.toFilePath()), '../static');
+
+/// Throws an error if any of the keys are missing from the object
+/// Parameters:
+///   obj - The object to check
+///   keys - The list of keys to check for
+/// throws Exception
```

Comment:
```dart
/// Throws an error if any of the keys are missing from the object
/// @param obj - The object to check
/// @param keys - The list of keys to check for
/// @throws Exception
```

In other templates, we use this format - can we match it here?

Reply:
Updated